pages/monsters/[id].vue

```diff
@@ -275,8 +295,12 @@ const monster = await useFetchArticle({
 const calcMod = (score) => Math.floor((score - 10) / 2);
 const formatMod = (mod) => (mod >= 0 ? '+' + mod.toString() : mod.toString());
 
+function uppercaseFirstLetter(string) {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
```

Comment:
Was going to move this into `/utils` but that folder seems to contains a bunch of Python utilities, so I wasn't sure if that folder was meant to be used for JS utility functions